tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/InlinePassImpl.scala

```diff
@@ -2,40 +2,80 @@ package at.forsyte.apalache.tla.pp.passes
 
 import at.forsyte.apalache.infra.passes.PassOptions
 import at.forsyte.apalache.io.lir.TlaWriterFactory
-import at.forsyte.apalache.tla.lir.storage.BodyMap
+import at.forsyte.apalache.tla.imp.findBodyOf
+import at.forsyte.apalache.tla.lir.{ModuleProperty, TlaModule, TlaOperDecl}
 import at.forsyte.apalache.tla.lir.transformations._
+import at.forsyte.apalache.tla.lir.transformations.standard.IncrementalRenaming
 import at.forsyte.apalache.tla.pp._
 import com.google.inject.Inject
+import com.typesafe.scalalogging.LazyLogging
 
 /**
- * A pass that expands operators and let-in definitions.
- *
- * @param options
- *   pass options
- * @param gen
- *   name generator
- * @param tracker
- *   transformation tracker
- * @param nextPass
- *   next pass to call
+ * A pass that expands operators and let-in definitions, as well as primitive-valued CONSTANTs.
  */
 class InlinePassImpl @Inject() (
```

Comment:
Do we need to document the new inliner in `docs/src/apalache/preprocessing.md`?

Reply:
There an open issue/PR for that #1567